cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -219,6 +230,128 @@ def _add_conditional_storage_widgets(
                 widgets_list.append(graph_widget)
         return widgets_list
 
+    def custom_pcluster_metric(self, metric_name, filter_pattern):
```

Comment:
`custom_pcluster_metric_filter` suits it better since we're creating and returning a MetricFilter